src/assets/export_to_cloud_dark.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe instead of `dark` a better name would be `selected`? When I see 'dark' it makes me think of the 'dark' themes that some websites have.

Reply:
I called it like that to differentiate it from the white colored version, maybe I should name them export_to_cloud_white and export_to_cloud_grey? BTW this version is only being used in the status table for the moment